server/controllers/questions.js

```diff
@@ -4,21 +4,22 @@ require('dotenv').config({ path: path.resolve(__dirname, '../../.env') });
 
 
 exports.getQuestions = (req, res) =>{
-
-  axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=71698', {
+  let apiGetQ = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/qa/questions?product_id=';
```

Comment:
If you're already using an .env file for your token, it's definitely worth not having the URL to the server on github